frontend/src/app/components/AppView/AppView.test.tsx

```diff
@@ -68,17 +96,15 @@ describe("AppView element", () => {
   })
 
   it("renders without crashing", () => {
-    const props = getProps()
-    const wrapper = shallow(<AppView {...props} />)
-
-    expect(wrapper).toBeDefined()
+    render(<AppView {...getProps()} />)
```

Comment:
?: I couldn't find confirmation either way, but if the render function fails, does the test automatically fail? Or is an expect statement always necessary?

Reply:
I think if there's an exception, this will throw the error. Though we are not testing that it returns stuff that will be defined.